app/models/schemas/workflow.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module Schemas
+  ##
+  # Schema for Donut administrative metadata
+  #
+  # @example applying the administrative schema
+  #
+  #   class WorkType < ActiveFedora::Base
+  #     include ::Hyrax::WorkBehavior
+  #     include ::Schemas::Administrative
+  #     include ::Hyrax::BasicMetadata
+  #   end
+  #
+  # @see ActiveFedora::Base
+  # @see Hyrax::WorkBehavior
+  module Workflow
+    extend ActiveSupport::Concern
+
+    included do
+      property :abstract, predicate: ::RDF::Vocab::DC.abstract, multiple: true do |index|
+        index.as :stored_searchable
+      end
+
+      property :box_name, predicate: ::Vocab::Donut.box_name, multiple: true do |index|
```

Comment:
The `box_name`, `box_number`, `folder_name`, and `folder_number` predicates should come from their equivalents at http://opaquenamespace.org/predicates e.g. the `box_name` predicate should be `::RDF::URI.new("http://opaquenamespace.org/ns/boxName")`